content/departments/engineering/teams/dev-experience/newsletter.md

```diff
@@ -8,6 +8,102 @@ To learn more about components of Sourcegraph's developer experience, check out
+This means that traces now export to an [OpenTelemetry Collector](https://docs.sourcegraph.com/admin/observability/opentelemetry) instance, which can then easily be configured to export data to a variety of backends - for example, in [s2](https://sourcegraph.sourcegraph.com) we are [currently exploring exporting to both Honeycomb, Google Cloud Traces, and Jaeger](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-managed@aa1ff34e9446c38c4376ecc29b3d75f4b4ae4676/-/blob/sg/red/otel-collector/config.yaml) to assess the available options.
+
+OpenTracing API calls are bridged to OpenTelemetry automatically, though we strongly recommend that everyone migrate to OpenTelemetry APIs (either `internal/trace` or `go.opentelemetry.io/otel`), which brings better API ergonomics and improved usage of `context.Context`. We've added a linter that forbids new imports of OpenTracing APIs, and added deprecation notices on internal APIs that should no longer be used.
+
+Additionally, the web app can now be instrumented with OpenTelemetry as well, with traces being sent to the frontend service which proxies it to the deployed OpenTelemetry Collector.
+
+To get started with testing out OpenTelemetry locally, refer to our [OpenTelemetry development docs](https://docs.sourcegraph.com/dev/how-to/opentelemetry_local_dev) and refer to our [tracing for site admins guidance](https://docs.sourcegraph.com/admin/observability/tracing).
+
+If you are interested in learning more about OpenTelemetry in general and the specifics of the engineering work required to make this happen, check out [this recording of a DevX team Q&A about OpenTelemetry](https://drive.google.com/drive/folders/1z_Uf_Stp3frDKB_UXFtenbQJdYB4LikO)!
+
+### Frontend news
+
+**`esbuild` for faster frontend builds**: Shoutout to @Nick Snyder who took it upon himself to get `esbuild` in a usable state for Sourcegraph. It doesn't work for everything yet but a few people have reported a markable improvement in sg startup time! You can enable it by running `DEV_WEB_BUILDER=`esbuild` sg start`. For more information on `esbuild` please see [the following docs.](https://docs.sourcegraph.com/dev/background-information/web/build#esbuild)
+
+**Documentation for analyzing the bundlesize check failure**: With more contributions focused on the core workflow improvements, the Frontend Platform team noticed an increased number of [bundlesize check](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize) [failures](https://github.com/sourcegraph/sourcegraph/runs/8094204910), and often the reason for it is not apparent. Valery added [a step-by-step guide for debugging root causes](https://docs.sourcegraph.com/dev/how-to/testing#analyzing-the-bundlesize-check-failure) which should help teams to triage this failure. The Frontend Platform team will be looking into automating these steps entirely next quarter!
```

Comment:
@jhchabran, I added one paragraph about [the FE docs update](https://github.com/sourcegraph/sourcegraph/pull/41164) for visibility 🙂